src/components/SearchModal/ImportToken.tsx

```diff
@@ -0,0 +1,82 @@
+import { TokenList } from '@uniswap/token-lists/dist/types'
+import React from 'react'
+import { Token, Currency } from 'dxswap-sdk'
+import styled from 'styled-components/macro'
+import { TYPE, CloseIcon } from '../../theme'
+import { AutoColumn } from '../Column'
+import { RowBetween } from '../Row'
+import { ButtonError } from '../Button'
+import { useAddUserToken } from '../../state/user/hooks'
+import { GoBackIcon, PaddedColumn } from './styleds'
+import { Text } from 'rebass'
+import { TokenWarningCard } from '../TokenWarningModal'
+
+const Wrapper = styled.div`
+  position: relative;
+  width: 100%;
+  overflow: auto;
+  background-color: ${({ theme }) => theme.bg1And2};
+`
+
+const BottomSectionContainer = styled.div`
+  background-color: ${({ theme }) => theme.bg1};
+  padding: 20px;
+`
+
+interface ImportProps {
+  tokens: Token[]
+  list?: TokenList
+  onBack: () => void
+  onDismiss: () => void
+  handleCurrencySelect?: (currency: Currency) => void
+}
+
+export function ImportToken({ tokens, list, onBack, onDismiss, handleCurrencySelect }: ImportProps) {
+  const addToken = useAddUserToken()
+
+  return (
+    <Wrapper>
+      <PaddedColumn gap="14px" style={{ width: '100%', flex: '1 1' }}>
+        <RowBetween>
+          <GoBackIcon onClick={onBack} />
```

Comment:
It's not the best practice to apply click events on images. How about wrap it in a button and use it there?